<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderImportDialog</class>
 <widget class="QDialog" name="OrderImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="importOrderButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>导入订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importOrderDetailButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>导入订单详情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createOrderButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>生成订单</string>
   </property>
  </widget>
  <widget class="QTableView" name="orderView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>591</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderFileEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderDetailFileEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
